src/features/operator/Fee/FeeWithdraw/index.tsx

```diff
@@ -31,7 +31,7 @@ interface IFormInputs {
 export const FeeWithdraw = (): JSX.Element => {
   const { lbtcUnit, network, currency } = useTypedSelector(({ settings }: RootState) => settings);
   const [form] = Form.useForm<IFormInputs>();
-  const [feeWithdrawAmount, setFeeWithdrawAmount] = useState<string>('');
+  const [feeWithdrawAmount, setFeeWithdrawAmount] = useState<string>('0');
```

Comment:
should't be converted to number before storing/retrieving in/from state? cc/ @Janaka-Steph

Reply:
I'm using a string because it really just reflects realtime the value of an input element as the user is typing it (string). It does get transformed properly afterwards (for the currency conversion).

Maybe I can rename the variable to feeWithdrawInputValue ¿?

Reply:
> Maybe I can rename the variable to feeWithdrawInputValue ¿?
agreed